src/EditorFeatures/Core.Wpf/Suggestions/SuggestedActionsSource.State.cs

```diff
@@ -26,7 +26,6 @@ protected sealed class State : IDisposable
 
                 // mutable state
                 public Workspace? Workspace { get; set; }
-                public int LastSolutionVersionReported;
```

Comment:
importantly, we remove this aspect of SuggestedActions which are quite obnoxious.  Today, if we're computing a lightbulb, and a workspace change happens, we issue an event saying we're stale.  teh editor then tears down the lightbulb leading to situations where users ask for hte lightbulb, see it computing, and then seeing it dissapear.  The premise here is that we hope that in the majority of times, we'll still have viable lightbulb items that can still be applied.

Note: users already encounter the case where they still try to invoke the lightbulb but see is fail out.  So this is still a possibility before/after, just that we should hopefully be encountering this less.  

I will add telemetry as well so we can see how many times this new logic helps, and how many times we still fail out completely.